client/verta/verta/monitoring/profiler.py

```diff
@@ -204,9 +204,9 @@ def profile_column(self, df, column):
             bins = self._bins[column]
         else:
             bins = self._bins
-        values, limits = self._np.histogram(df[column], bins=bins)
-        values = [v.item() for v in values]
-        limits = [lim.item() for lim in limits]
+        values, limits = self._np.histogram(df[column].dropna(), bins=bins)
+        values = values.tolist()
+        limits = limits.tolist()
```

Comment:
I was running into an issue where the `np.array` `limits` sometimes contained Python builtin `float`s rather than NumPy values (so `item()` fails).

As far as I can tell from [NumPy's docs and examples](https://numpy.org/doc/stable/reference/generated/numpy.histogram.html), `values` and `limits` will always be 1-D arrays so it should be safe to use `array.tolist()` rather than `element.item()`.